spec/lang/well-formed.md

```diff
@@ -274,23 +295,23 @@ impl ValueExpr {
                 match operator {
                     Int(_int_op) => {
                         let Type::Int(int_ty) = operand else {
-                            throw!();
+                            throw_ill_formed!("ValueExpr::UnOp::Int: invalid operand");
```

Comment:
```suggestion
                            throw_ill_formed!("UnOp::Int: invalid operand");
```
`UnOp` already implies it's a value expression, let's not be unnecessarily verbose. (Applies everywhere. I specifically an example with UnOp in my previous review to avoid this. ;)

Reply:
Removed the verbosity everywhere. Now the maximum depth is `a::b:` and it works out fine. 👍🏼